HW5/src/Main/app.ts

```diff
@@ -1,16 +1,17 @@
-import { modal } from "./modal.js";
-import { showModal } from "./modalActions.js";
-import { renderTaskList } from "./todoItem.js";
-import { search } from "./search.js";
-import { weatherWidget } from "./weatherWidget.js";
-import { daylyTaskModal, isShownToday } from "./daylyTaskModal.js";
+import { modal } from "../modals/modal";
```

Comment:
You have an error here
Already included file name '...' differs from file name '...'
So it's the problem in the naming: you have Modals, Popup and etc folders, but you trying to use it as modals, popup, ...